docs/conf.py

```diff
@@ -5,7 +5,11 @@
 # -- stdlib imports ------------------------------------------------------------
 import os
 import warnings
-from datetime import UTC, datetime
+
+# from datetime import UTC, datetime #Feb2025 - comment out
+from datetime import datetime, timezone  # Feb2025
+
+UTC = timezone.utc  # Define UTC manually for Python 3.10 compatibility Feb2025
```

Comment:
We dropped Python 3.10 support in #330, so we can use the [newer way](https://docs.python.org/3/whatsnew/3.11.html#datetime) of defining `UTC`.